tensorflow/python/ops/array_ops.py

```diff
@@ -2033,25 +2033,25 @@ def edit_distance(hypothesis, truth, normalize=True, name="edit_distance"):
   For example, given the following input:
 
   ```python
-  # 'hypothesis' is a tensor of shape `[2, 1]` with variable-length values:
```

Comment:
I think the original comments are actually correct, though a bit misleading. The idea is that hypothesis represents a dense [2, 1] tensor where each value is variable length. This is represented as a Sparse 3d tensor (3rd dimension represents the sequence dimension). 

Perhaps writing the comment as "'hypothesis' represents" instead of "'hypothesis' is" would be helpful?

Reply:
Sorry for the misunderstanding that. And there are two commas lost in the construction of SparseTensor?

Reply:
Yes, that would be a welcome fix

Reply:
OK，I changed the code back except for two commas